README.MD

```diff
@@ -58,7 +56,7 @@ You can try Brizy without installing it. Simply use one of the demos to deploy i
 
 You can register your own arbitrary custom React components for use as building blocks within Brizy.
 
-[Learn more about code components.](https://brizy.io)
+[Learn more about code components.](https://builder-docs.brizy.io/docs/development/creating-your-first-addon)
```

Comment:
maybe link should be only `/docs/development/creating-your-first-addon` ?
like in other files